packages/flutter/test/material/drawer_test.dart

```diff
@@ -106,4 +106,52 @@ void main() {
 
     semantics.dispose();
   });
+
+  testWidgets('Drawer scrimDrawerColor test', (WidgetTester tester) async {
+    const Key containerKey = Key('container');
```

Comment:
You can simplify this setup a bit, you don't need the container key, and in fact you can just have the drawer have no child since you're not testing that functionality.

Reply:
fixed! :D @rami-a